### PR TITLE
[BACKLOG-35812] - moved excel steps from core to plugin

### PR DESCRIPTION
**Pull request by sVarma1992**
[BACKLOG-35812] - moved excel steps from core to plugin

### CODE REVIEW

File: plugins/excel/ui/src/test/java/org/pentaho/di/ui/trans/steps/excelinput/ExcelInputDialogTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
+///*! ******************************************************************************
+// *
+// * Pentaho Data Integration
```
Comment (peterrinehart) [inline plugins/excel/ui/src/test/java/org/pentaho/di/ui/trans/steps/excelinput/ExcelInputDialogTest.java:3]: Why is this entire test commented out?

**Review by peterrinehart — CHANGES_REQUESTED** [review]
Any java files with code changes need their copyright date updated in the code header.  If the file was only moved, I don't think it's necessary to update the copyright date.

Please also address additional comments in this review.

File: plugins/excel/ui/src/test/java/org/pentaho/di/ui/trans/steps/excelinput/ExcelInputDialogTest.java (hunk 1 above)
Comment (sVarma1992) [reply plugins/excel/ui/src/test/java/org/pentaho/di/ui/trans/steps/excelinput/ExcelInputDialogTest.java]: Commented while developing and by mistake forgot to uncomment while pushing changes

**Review by peterrinehart — APPROVED** [review]

**Review by peterrinehart — CHANGES_REQUESTED** [review]
Please squash the two commits in this PR into one, and I can merge it.

**Review by peterrinehart — APPROVED** [review]

**Pull request merged**